python/ray/tune/experiment.py

```diff
@@ -136,7 +119,7 @@ def __init__(self,
             # Most users won't run into this as `tune.run()` does not pass
             # the argument anymore. However, we will want to inform users
             # if they instantiate their `Experiment` objects themselves.
-            raise ValueError(
```

Comment:
Not related to this PR. But what is our depreciation cycle? This error was there since 11 months ago. Can we remove `loggers` from `__init__`?

Reply:
Right, we can probably remove it from this class as it raised an error before